src/theory/bags/normal_form.h

```diff
@@ -0,0 +1,49 @@
+/*********************                                                        */
+/*! \file normal_form.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Kshitij Bansal, Andrew Reynolds, Tim King
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2020 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Normal form for bag constants.
+ **
+ ** Normal form for bag constants.
+ **/
+
+#include "cvc4_private.h"
+
+#ifndef CVC4__THEORY__BAGS__NORMAL_FORM_H
+#define CVC4__THEORY__BAGS__NORMAL_FORM_H
+
+namespace CVC4 {
+namespace theory {
+namespace bags {
+
+class NormalForm
+{
+ public:
+  /**
+   * Returns true if n is considered a to be a (canonical) constant bag value.
+   * A canonical bag value is one whose AST is:
+   *   (disjoint-union (bag-pair e1 n1) ...
+   *        (disjoint-union (bag-pair e_{n-1} n_{n-1}) (bag-pair e_n n_n))))
+   * where c1 ... cn are constants and the node identifier of these constants
+   * are such that:
+   *   c1 > ... > cn.
+   * Also handles the corner cases of empty bag and singleton bag.
+   */
+  static bool checkNormalConstant(TNode n)
+  {
+    // ToDo: complete this function
```

Comment:
Do not write "ToDo", always use "TODO".  Also, TODO should be linked to an issue. Can you open a project issue and mention the number by all the TODO you introduce?

Reply:
Done. [223](https://github.com/CVC4/cvc4-projects/issues/223). There is no project for bags, so I added it sets. 
Please can you create a project for bags. I don't have permissions.